pkg/webui/console/containers/schedule-delay-input/index.js

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+
+import PropTypes from '../../../lib/prop-types'
+
+import Field from '../../../components/form/field'
+import DurationInput from './schedule-delay-input'
+
+const ScheduleDelayField = function({
```

Comment:
Unless I'm not missing something here, this whole construct is unnecessary.
You could simply use the `<Field />` component directly and pass the props. Wrapping only makes sense if you need something from the store to be present in the field.